lib/linguist/languages.yml

```diff
@@ -1455,6 +1455,15 @@ Fantom:
   tm_scope: source.fan
   ace_mode: text
   language_id: 110
+Faust:
+  type: programming
+  color: "#c37240"
+  extensions:
+  - ".dsp"
+  - ".lib"
```

Comment:
Hm, I missed that file extension. You'll need to provide a search query for that one too. We'll have to be particularly careful for that one, it's likely to conflict with other languages.

Reply:
Oh yes, for the ".lib" file suffix:
https://github.com/search?p=2&q=extension%3Alib+%22declare+name%22&type=Code

It's nothing special compared to ".dsp", it's just an optional naming which hints at the file being a library of functions.

What steps does it take to disambiguate this one?

Reply:
Piouf, that one is huge: [`extension:lib NOT "declare name"`](https://github.com/search?o=desc&q=extension%3Alib+NOT+%22declare+name%22&s=indexed&type=Code). Are you sure you don't want to add it in a second pull request? There seem to be a lot of different languages there and it's going to take us a while to sort them out. It might be better to get `.dsp` in first, and then address `.lib`.

Reply:
It is perfectly fine with me, I removed `.lib` in the commit f16e596f.